app/src/main/java/nerd/tuxmobil/fahrplan/congress/designsystem/colors/ColorScheme.kt

```diff
@@ -58,9 +58,33 @@ data class ColorScheme(
     val onTertiaryFixedVariant: Color,
 
     // Custom colors
-    val topAppBarContainer: Color,
+    val listItem: Color,
```

Comment:
I think this name should be more specific.

- What is the color of a list item? The background color? The text color?
- Does this apply to all list items? Or is it specific to e.g. schedule list items?